perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -187,45 +227,51 @@ def StartCluster(seed_vm, vms):
     vms: list of VirtualMachines. VMs *other than* seed_vm which should be
       started.
   """
-  vm_count = len(vms) + 1
-
-  # Cassandra setup
-  logging.info('Starting seed VM %s', seed_vm)
-  Start(seed_vm)
-  logging.info('Waiting %ds for seed to start', NODE_START_SLEEP)
-  time.sleep(NODE_START_SLEEP)
-  for i in xrange(5):
-    if not IsRunning(seed_vm):
-      logging.warn('Seed %s: Cassandra not running yet (try %d). Waiting %ds.',
-                   seed_vm, i, NODE_START_SLEEP)
-      time.sleep(NODE_START_SLEEP)
+  if seed_vm.OS_TYPE != JUJU:
```

Comment:
Instead of 
```
if vm.OS_TYPE != JUJU:
  rest of function body
```
Maybe do
```
if vm.OS_TYPE == JUJU:
  return
rest of function body
```
This would make it easier to keep track of indent levels (this would be most helpful here since it's the longest function, but it might be nice to make the other functions match)